### PR TITLE
Allow running a subset of the basic unit tests without launching a MIDlet

### PR DESCRIPTION
**Pull request by marco-c**
Sometimes it's useful to run tests without launching a MIDlet.
Launching a MIDlet is a quite complex process, so if you're debugging a low-level feature the chances that you can't even start the current test harness are high.

### CODE REVIEW

File: tests/Harness.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import gnu.testlet.*;
+
+import com.sun.cldchi.jvm.JVM;
+import javax.microedition.lcdui.Display;
+import javax.microedition.lcdui.Form;
+import javax.microedition.midlet.*;
+import java.lang.Exception;
+import java.util.Vector;
+
+public class Harness extends TestHarness {
```
Comment (mykmelez) [inline tests/Harness.java]: Nit: it's strange that the name of this class, *Harness*, is more generic than the name of the superclass it extends, *TestHarness*. I would give it a more specific name than that class, like *RunTestsHarness* or *j2mejsTestHarness*.

File: tests/gnu/testlet/MIDletTestlet.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package gnu.testlet;
+
+public interface MIDletTestlet extends Testlet {
```
Comment (mykmelez) [inline tests/gnu/testlet/MIDletTestlet.java]: Is there something you plan to put in *MIDletTestlet*? It isn't clear why this is necessary when it doesn't actually change or add to the *Testlet* class.

Comment (marco-c) [inline tests/gnu/testlet/MIDletTestlet.java]: I added this interface to be able to simply grep to find classes to add to the MIDletTestlets list.

Comment (mykmelez) [inline tests/gnu/testlet/MIDletTestlet.java]: Ok, that's fine. ;-)

File: tests/Harness.java (hunk 1 above)
Comment (marco-c) [inline tests/Harness.java]: Renamed Harness to j2mejsTestHarness in a884190.

**Pull request merged**